index.html

```diff
@@ -97,32 +116,192 @@
+    					<a href="">
+	    					<div class="icons-mobile d-flex justify-content-center align-items-center">
+	    						<div class="background-icon d-flex justify-content-center align-items-center">
+    								<img class="fix-icon" src="img/fix-icon.png">
+    							</div> 
+	    					</div>   			
+		    				<div class="nav-text-con">
+			    				<div class="gallery-nav-text d-flex flex-column justify-content-center">
+			    					<h2>פזומט fix</h2>
+				    				<p>כפתורי דמה אתר פז ויחליככגה לויהכךלנינל</p>
+			    				</div>
+		    				</div>
+	    				</a>
+    				</li>
+    				<li class="d-flex justify-content-center">
+    					<a href="">
+	    					<div class="icons-mobile d-flex justify-content-center align-items-center">
+	    						<div class="background-icon d-flex justify-content-center align-items-center">
+    								<img src="img/service-icon.png">
+    							</div>
+	    					</div>
+		    				<div class="nav-text-con">
+			    				<div class="gallery-nav-text d-flex flex-column justify-content-center">
+			    					<h2>מובילים בשירות</h2>
+				    				<p>כפתורי דמה אתר פז ויחליככגה לויהכךלנינל</p>
+			    				</div>
+		    				</div>
+	    				</a>
+    				</li>
+    			</ul>
+    		</nav>
+		</div>
+	</section>
 
-
-
-    	<!-- TOP SECTION -->
-    	<section>
-
-    	</section>
-    </div>
+	<!-- BOTTOM SECTION -->
+	<section class="bottom-section container">
+		<div class="mishpatiot-player"><h1>--bar1--</h1></div>
+		<div class="logo-gallery"><h1>--bar2--</h1></div>
+		<footer class="links">
+			<div class="top-footer container d-flex flex-lg-row flex-column">
+				<div class="top-footer-mobile d-lg-none">
+					<div>
+						<a href=""><spen>עמוד הבית</spen></a>
+						<a class="odot" href=""><spen>אודות</spen></a>
+						<a class="last-in-row" href=""><spen>אחריות תאגידית</spen></a></div>
+					<div>
+						<a href=""><spen>חברות הקבוצה</spen></a>
+						<a class="last-in-row" href=""><spen>התחנות שלנו</spen></a></div>
+					<div>
+						<a href=""><spen>קשרי משקיעים</spen></a>
+						<a class="last-in-row" href=""><spen>צור קשר</spen></a></div>
+				</div>
+				<div class="footer-icon-nav-mobile d-lg-none d-flex flex-column">
+					<div class="top d-flex flex-row">
+						<div class="top-t">
+							<img src="img/icon-1.svg" class="icon1">
+							<div class="icon-h">מועדון נאמנות</div>
+						</div>
+						<div>
+							<img src="img/icon-2.svg" class="icon2">
+							<div class="icon-h">איתור תחנת דלק</div>
+						</div>
+					</div>
+					<div class="bottom d-flex flex-row">
+						<div class="top-b">
+							<img src="img/icon-4.svg" class="icon4">
+							<div class="icon-h">נתוני מנייה</div>
+						</div>
+						<div>
+							<img src="img/icon-3.svg" class="icon3">
+							<div class="icon-h">כנסו ושתפו</div>
+						</div>
+					</div>
+				</div>
+				<div class="left-footer d-lg-flex row">
+					<ul class="d-flex flex-lg-row flex-column col align-items-center justify-content-around">
+						<a href="">
```

Comment:
# Improve the `top-footer`
There's a better way to style the _container_: `top-footer` and it's _children_:  `left-footer` & `right-footer`. The way it is right now looks good but if you shrink the screen blow `1320px` the elements have no way to resize and the layout breaks.

## Here's how to fix it...

### 1 
This selector: `.bottom-section footer .top-footer .left-footer` has these styles:
```
flex-grow: 1;
margin-right: 120px;
```
Remove them. Let the content of each side (i.e. the left and right containers) decide their width. There's no reason to give them their own width. Also **don't use margin in cases where you can let flex-box space stuff for you** (for example `space-between`).

### 2
This selector `.bottom-section footer .top-footer .left-footer ul a` has:
```
adding: 26.5px 0 26.5px 25px;
```
Change this to:
```
adding: 26.5 25px;
```
There's no reason to set the `padding-right` property to 0 here, it just creates asymmetry.

### 3
Remove the `col` class from `right-footer`. **When we have a case in the design where elements are located on each side of the container we want to make the right and left side get the width from the content and use `flex-box` to position them on either side**.

### 4
At this point the left and right containers are aligning to the left so you can simply use `flex-box` to space between them by adding `justify-content-around` to the `top-footer` element

## Result
Now if you resize the screen you'll see that the elements are not breaking until the get to the point when there is really not enough space to contain them and not before that.